src/main/java/cashleh/commands/Command.java

```diff
@@ -12,17 +12,8 @@ public class Command {
     protected Logger logger = Logger.getLogger("CommandLogger");
     protected Level loggerLevel = Level.OFF;
 
-    private int index;
-
-    public Command(int index) {
-        this.index = index;
-    }
-
     public Command() {
-    }
-
-    public int getIndex() {
-        return this.index;
+        this.logger.setLevel(loggerLevel);
```

Comment:
Nice way of disabling logging. Maybe in the next version we can also create a .txt file containing all the logging information separately.